src/components/Newsletter/Newsletter.js

```diff
@@ -1,12 +1,14 @@
 import firebase from 'firebase';
 import React from 'react';
-import { StyledNewsletter, InputHolder } from './Newsletter.style';
+import { StyledNewsletter, InputHolder, InputError } from './Newsletter.style';
 import Chip from '@material-ui/core/Chip';
 import Autocomplete from '@material-ui/lab/Autocomplete';
 import TextField from '@material-ui/core/TextField';
 import { getLocationNames } from 'common/locations';
 import { getFirebase } from 'common/firebase';
 
+const EMAIL_REGEX = /^([a-zA-Z0-9_\.\-])+\@(([a-zA-Z0-9\-])+\.)+([a-zA-Z0-9]{2,4})+$/;
```

Comment:
Did you make this yourself or copy it from somewhere?  If the latter, can you site the source?

I think email regexps are kinda' fragile... Like this will probably break with unicode, which is kinda' maybe supported in emails?  Not sure.  It's tempting to just use a suuuuper simple regex like https://stackoverflow.com/a/742455 which will catch the most common typos.  🤷 

FWIW- I also don't understand the `+` at the end...  It seems like `([a-zA-Z0-9]{2,4})` says you must have 2-4 letters or digits after the final dot, but then the `+` says you can have more than one of those sets of 2-4 letters or digits...  So it'd be equivalent and simpler to just have `[a-zA-Z0-9]+` (i.e. an arbitrary number of letters/digits) which makes sense to me (TLDs can definitely be longer than 4 chars these days (e.g. `.design`))

Reply:
ahh yeah i did copy it from somewhere (if only my personal regexs were so good and sorry ya i had another one that was just {2, }. Happy for the symbol one (maybe `^[^@\s]+@[^@\s\.]+\.[^@\.\s]+$` since it's a lil more specific

Reply:
/^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/ ended up using this one from https://emailregex.com/

Reply:
okay once again https://ui.dev/validate-email-address-javascript/ using that now simpler and less confusing